updates/3/updates.sql

```diff
@@ -49,4 +49,38 @@ INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingec
 INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("STEP", "Y", "Step Finance", "STEP", 1619335055, NULL, "step-finance", "", "STEP"); INSERT INTO common_asset_details(asset_id, forked) VALUES("STEP", NULL);
 *
 INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x4e3FBD56CD56c3e72c1403e103b45Db9da5B9D2B", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x4e3FBD56CD56c3e72c1403e103b45Db9da5B9D2B", "C", "Convex Token", "CVX", 1621242525, NULL, "convex-finance", "", "0x4e3FBD56CD56c3e72c1403e103b45Db9da5B9D2B");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xC08512927D12348F6620a698105e1BAac6EcD911", 6, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xC08512927D12348F6620a698105e1BAac6EcD911", "C", "GMO JPY", "GYEN", 1576043730, NULL, "gyen", "GYEN", "0xC08512927D12348F6620a698105e1BAac6EcD911");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xB8E2e2101eD11e9138803cd3e06e16dd19910647", 2, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xB8E2e2101eD11e9138803cd3e06e16dd19910647", "C", "ArdCoin", "ARDX", 1566984871, NULL, "ardcoin", "ARDX", "0xB8E2e2101eD11e9138803cd3e06e16dd19910647");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x6F919D67967a97EA36195A2346d9244E60FE0dDB", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x6F919D67967a97EA36195A2346d9244E60FE0dDB", "C", "Blockcloud", "BLOC", 1554704948, NULL, "blockcloud", "BLOC", "0x6F919D67967a97EA36195A2346d9244E60FE0dDB");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xfAE4Ee59CDd86e3Be9e8b90b53AA866327D7c090", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xfAE4Ee59CDd86e3Be9e8b90b53AA866327D7c090", "C", "CPChain", "CPC", 1516127905, NULL, "cpchain", "CPC", "0xfAE4Ee59CDd86e3Be9e8b90b53AA866327D7c090");
```

Comment:
This is already there too. Exact same address.

```
sqlite> SELECT * from assets WHERE symbol="CPC";
_ceth_0xfAE4Ee59CDd86e3Be9e8b90b53AA866327D7c090|C|CPChain|CPC|1516060800||cpchain|CPCH|0xfAE4Ee59CDd86e3Be9e8b90b53AA866327D7c090
CPC-2|B|Capricoin|CPC|1437350400||capricoin|CPC|CPC-2
```

Update won't work.

Reply:
The cryptocompare is wrong. `CPCH` -> `CPC` needs to be an update